### ISSUE TITLE
Code question

### ISSUE THREAD

**Issue by llchen9426** [fhn98/DaC#1]
Hello, the source code of this article, I encountered some problems in the process of implementation, in the training of ERM model (the first process of training), the error message: The mask file of the image data needs to be found (the npy format of the corresponding file cannot be found, that is, the mask). According to the thesis, the mask is not used in the ERM model training process, but the mask is the second training process. I modified the code but failed, has anyone encountered this problem? Thank you!

**Comment by fhn98** [comment]
Thank you for pointing this out! You’re right — the ERM phase does not require masks. This has been fixed in the latest [commit](https://github.com/fhn98/DaC/commit/cb4f76f0a8533663f2c29e6a208b910dfc6ce233). Let us know if you run into anything else!

**Comment by fhn98** [comment]
Closing this issue for now, but feel free to reopen it if anything comes up. Thanks again for catching this!

**Comment by llchen9426** [comment]
Hello, I tried to implement your updated code, but it still didn't run successfully, I'm using a 3090 graphics card, and the run reports an error: an error occurred while performing the matrix multiplication (wrong format or dimension of the input data), and I see this code in the train.py
if args.experiment == 'ERM':
        print('Getting Dataloaders...')
        trainloader, valloader, testloader = get_loaders(args.dataset, path=args.data_path, mask_path=args.mask_path,
                                                         batch_size=args.batch_size, get_mask=False)
        print('Dataloaders prepared')
        trainable_parameters = model.parameters()

Does the mask_path in this need to be changed to None, I tried and it still failed, can you run the full code you uploaded on GitHub on your end? I'm not sure if it's my device. Hope to get your reply, thanks!
